projects/mysadna/src/app/app-routing.module.ts

```diff
@@ -9,6 +9,7 @@ import {
 } from '@/components';
 
 const routes: Routes = [
+  { path: '', redirectTo: '/projects', pathMatch: 'full' },
```

Comment:
Great!

Reply:
But let's keep it in one style:
s/ `/projects` / `projects`

Reply:
I don't understand what you mean.

Reply:
Look at other routes, which you implemented before:
```
path: 'contacts'
path: 'user-details/:userId'
path: 'projects'
```
They doesn't have `/` at the beginning. So your new route should have it too. Add you should add `/` to all routes

Reply:
Now I understand.